enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -689,9 +689,6 @@ def can_redeem(self, lms_user_id, content_key, skip_customer_user_check=False):
         # learner not associated to enterprise
         if not skip_customer_user_check:
             enterprise_user_record = self.enterprise_user_record(lms_user_id)
-            if not enterprise_user_record:
```

Comment:
Were there any unit tests around this condition?  If not, obviously we don't need to add them now, but it may point to testing gaps around `can_redeem` that should be addressed at some point.